tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py35, py36
+envlist = py39
```

Python 3.9 is pretty old at this point and hit the end of support 2 years ago, although it's common on the LTS Linux distributions. I'd go up to 3.12 at this point.

```suggestion
envlist = py39,py310,py311,py312
```